buildpipeline/DotNet-CoreFx-Trusted-OSX.json

```diff
@@ -291,6 +291,14 @@
       "value": null,
       "isSecret": true
     },
+    "CloudResultsAccessToken": {
```

Comment:
The parameter is actually named OutputCloudResultsAccessToken

Reply:
thanks for the catch! fixed! and found out that token was also needed in Windows!